firewall-examples.md

```diff
@@ -15,28 +13,23 @@ $ sdc-enablemachinefirewall &lt;machine-id&gt;
 </div></div>
 
 
-<p>Recall that for instances that have Cloud Firewall feature enabled, the default policy is always:</p>
+Recall that for instances that have Cloud Firewall feature enabled, the default policy is always:
 
-<ul>
-	<li>allow incoming ping requests</li>
-	<li>block all other incoming traffic</li>
-	<li>allow all outgoing traffic</li>
-</ul>
+	- allow incoming ping requests
```

Comment:
Tabbing these in forces markdown to format them like a code block, rather than bulleted content:

![screen shot 2015-06-15 at 9 50 49 pm](https://cloud.githubusercontent.com/assets/1709697/8176000/adf0eaf6-13a8-11e5-8799-240a64c699d6.png)

Line these up on the left edge and they'll format correctly.